docs/ballerina-by-example/examples/websocket-proxy-server/websocket-proxy-server.bal

```
@@ -25,17 +25,17 @@ service<http:WebSocketService> SimpleProxyServer bind serviceEndpoint {
 
     onTextMessage (endpoint ep, http:TextFrame frame) {
         endpoint http:WebSocketClient clientEp = getAssociatedClientEndpoint(ep);
-        clientEp -> pushText(frame.text);
+        _ = clientEp -> pushText(frame.text);
```

In examples, we should not ignore the error.